### PR TITLE
Connect vscode to remote wb

### PR DESCRIPTION
**Pull request by tokalevasant**
Added documentation on connecting to RHOAI workbench kernel from local VS Code

### CODE REVIEW

File: docs/odh-rhoai/connect-vscode-to-rhoai-wb.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Connect to RHOAI Workbench Kernel from local VS Code
+
+Some customers have expressed their desire to work directly on their local IDE and execute the Jupyter notebook(s) using the kernel on remote workbench running on RHOAI. Most IDEs provide connection to a remote Kernel as a standard feature. However, this standard feature does not work with RHOAI because of the way authentication to workbench is set up in RHOAI<br/>
```
Comment (guimou) [inline docs/odh-rhoai/connect-vscode-to-rhoai-wb.md:3]: Could you rephrase to remove "customers"? All the recipes here can also be used on upstream ODH, so we try to keep this agnostic. Mentioning RHOAI is ok though.

File: docs/odh-rhoai/connect-vscode-to-rhoai-wb.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+The standard feature of most IDEs to connect to remote kernel uses token based authentication. Workbench pods running on RHOAI contain a OAuth proxy container that sits in front of the workbench container and handles the authentication of the user connecting to the workbench. This container uses Openshift Authentication mechanism and is not compatible with the standard connection feature of most IDEs.
```
Comment (guimou) [inline docs/odh-rhoai/connect-vscode-to-rhoai-wb.md:4]: OAuth proxy will be replaced soon by Authorino. I would say "... contain an authentication mechanism that sits in front of the workbench..." to handle whatever version people are running.

File: docs/odh-rhoai/connect-vscode-to-rhoai-wb.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- From terminal on your laptop/desktop login to openshift.
+    ![Login to openshift](img/login-to-ocp.png)
+
+- Switch to your data science project
+    ![Switch project](img/switch-to-project.png)
+
+- Start port-forwarding to your workbench pod
+    * List all the pods in your project. The pod running your workbench is named using the name of your workbench in RHOAI. e.g. `mywb-0` if your workbench name is `mywb`.
+    * Enable port-forwarding to your workbench pod. You need to forward to the port the pod is listening on. It is usually 8888 for RHOAI workbench. You can find this port from the service in your project with name same as your workbench.
+    ![Port-forward to wb pod](img/port-forwarding.png)
```
Comment (guimou) [inline docs/odh-rhoai/connect-vscode-to-rhoai-wb.md:27]: Have you tested it as a standard user? I remember having issues with port-forwarding, but I seem to remember it was because I was port-forwarding to the Service and not the Pod directly, which requires to have "list" rights on Services for the user account. Or something like that.

**Review by guimou — COMMENTED** [review]
Nice addition! I was doing this like two years ago, but never took the time to document it. Thanks a lot!
Minor editions needed if you can.

File: docs/odh-rhoai/connect-vscode-to-rhoai-wb.md (hunk 2 above)
Comment (tokalevasant) [reply docs/odh-rhoai/connect-vscode-to-rhoai-wb.md:4]: Fixed

File: docs/odh-rhoai/connect-vscode-to-rhoai-wb.md (hunk 3 above)
Comment (tokalevasant) [reply docs/odh-rhoai/connect-vscode-to-rhoai-wb.md:27]: I tested with a normal user with edit permission to the project and I was able to port-forward to the pod and connect from my vs code.
